internal/cache/controller_obj_cache.go

```diff
@@ -2652,12 +2652,27 @@ func checkPublicCloud(client *clients.AviClient) bool {
 		// Handle all public cloud validations here
 		networkName := lib.GetNetworkName()
 		if networkName == "" && lib.GetCloudType() != lib.CLOUD_GCP {
-			// networkName is required param for AWS and Azure Clouds
+			if lib.GetCloudType() == lib.CLOUD_AWS {
+				//check for vipNetworkList if networkName not present for AWS Clouds
+				utils.AviLog.Infof("networkName not specified")
+				utils.AviLog.Infof("Checking for vipNetworkList as cloud is AWS")
```

Comment:
these two can probable be merged into just one Info log.

Reply:
merged into single log